### ISSUE TITLE
template string highlighting not working

### ISSUE THREAD

**Issue by q42jaap** [Microsoft/vscode#1020]
The template string highlighting is broken (it's announced to be working in 0.5.0: http://blogs.msdn.com/b/vscode/archive/2015/07/06/vs-code-es6.aspx).

My jsconfig.json file:
```
{
  "compilerOptions": {
    "target": "ES6",
    "module": "commonjs"
  }
}
```

backtick.js:
```
var x = null;
var y = `Hello ${x} World/*`
var z = null;
```

placed in the same directory and then open visual studio code with:
`open .`

using the following line in `.bash_profile`:
```bash
code () { VSCODE_CWD="$PWD" open -n -b "com.microsoft.VSCode" --args $* ;}
```

This results in the second line to be green (because of `/* which starts a block comment), also the last line is compeletely green as if it's a block comment...

If I add `var sum = (var1, var2) => var1 + var2` as the first line in backtick.js, is highlighted correctly. If I remove jsconfig.json, it starts complaining. For me this is a check to see if the options jsonfig.json are correct and picked up by Code.

**Comment by egamma** [comment]
Since the blog post we have changed the coloring to textmate grammar and in December we are now switching to the TypeScript grammar.

Assigning to @aeschli to verify whether this now working properly.

**Comment by aeschli** [comment]
Verified that this got fixed by #133.
